lib/pay/stripe/billable.rb

```diff
@@ -47,7 +52,7 @@ def create_stripe_charge(amount, options = {})
       # Handles Billable#subscribe
       #
       # Returns Pay::Subscription
-      def create_stripe_subscription(name, plan, options = {})
+      def create_stripe_subscription(name, plan, options = {}, idempotency_key = SecureRandom.uuid)
```

Comment:
What if we have the idempotency key as part of options and remove it when passed in like we do with quantity? That way it doesn't change the method signature and we can seamlessly remove it like we do with the `quantity`